lib/ansible/modules/cloud/amazon/cloudwatchlogs_log_group_info.py

```diff
@@ -115,13 +117,22 @@ def main():
     region, ec2_url, aws_connect_kwargs = get_aws_connection_info(module, boto3=True)
     logs = boto3_conn(module, conn_type='client', resource='logs', region=region, endpoint=ec2_url, **aws_connect_kwargs)
 
-    desc_log_group = describe_log_group(client=logs,
-                                        log_group_name=module.params['log_group_name'],
-                                        module=module)
     final_log_group_snake = []
+    group_next_token = None
 
-    for log_group in desc_log_group['logGroups']:
-        final_log_group_snake.append(camel_dict_to_snake_dict(log_group))
+    while True:
```

Comment:
It looks like there's already a paginator for this https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/logs.html#CloudWatchLogs.Paginator.DescribeLogGroups
To get the full list of log groups with the paginator you can do something like `paginator.paginate(**kwargs).build_full_result()`

Reply:
@s-hertel 
Thank you for that hint. I did change it to use the paginator.